sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_sync_token.py

```diff
@@ -40,27 +41,31 @@ def __str__(self):
     @classmethod
     def from_sync_token_string(cls, sync_token):
         try:
-            position = sync_token.index(';sn=')
-            sequence_number = int(sync_token[position+4:])
+            position = sync_token.index(";sn=")
+            sequence_number = int(sync_token[position + 4 :])
             id_value = sync_token[:position]
-            position = id_value.index('=')
+            position = id_value.index("=")
             token_id = id_value[:position]
-            value = id_value[position+1:]
+            value = id_value[position + 1 :]
             return SyncToken(token_id, value, sequence_number)
         except ValueError:
             return None
 
+
 class SyncTokenPolicy(SansIOHTTPPolicy):
     """A simple policy that enable the given callback
     with the response.
 
     :keyword callback raw_response_hook: Callback function. Will be invoked on response.
     """
-    def __init__(self, **kwargs): # pylint: disable=unused-argument,super-init-not-called
+
+    def __init__(
+        self, **kwargs
+    ):  # pylint: disable=unused-argument,super-init-not-called
```

Comment:
This one's interesting -- I don't know if pylint would raise an error if the comment makes the line too long, so I'm not sure this is necessary. It looks kind of awkward but it's also not a biggie

Reply:
One of the disables was unnecessary, removing it makes it look cleaner.